incubator/prometheus/templates/alertmanager-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: alertmanager
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "alertmanager.fullname" . }}
+spec:
+  ports:
+    - port: {{ default 80 .Values.alertmanager.httpPort }}
+      name: {{ default "http" .Values.alertmanager.httpPortName | quote }}
+      protocol: TCP
+      targetPort: 9093
+  selector:
```

Comment:
Is it worth having type LoadBalancer available as an configurable option in the case where you don't have ingress controllers and want to expose the service externally?